cmd/dra-example-kubeletplugin/state.go

```diff
@@ -380,3 +400,56 @@ func GetOpaqueDeviceConfigs(
 
 	return resultConfigs, nil
 }
+
+func (s *DeviceState) buildDeviceStatus(res *resourceapi.DeviceRequestAllocationResult) *resourceapply.AllocatedDeviceStatusApplyConfiguration {
```

Comment:
Does `res` need to be a pointer, or can we save the extra syntax and theoretical nil-dereference by making this a plain value? Not a blocking issue, but I may refactor this when I integrate with #129.

Reply:
fixed.